releases/net471/KnownIssues/479874-WPF Touch Stops Working After Prolonged Use of Popups.md

```diff
@@ -0,0 +1,12 @@
+## WPF Touch Stops Working After Rapid/Prolonged Use Of Popups
+
+### Symptoms
+On a touch or stylus enabled machine, WPF applications may see a loss of touch or stylus after rapid or prolonged use of Popups (e.g. ComboBoxes, DropDown Menus)
```

Comment:
stylus enabled --> stylus-enabled
Popups --> popups
need a period to end the sentence